grpc/pb-marshalling_test.go

```diff
@@ -52,11 +52,13 @@ func TestChallenge(t *testing.T) {
 	var jwk jose.JSONWebKey
 	err := json.Unmarshal([]byte(JWK1JSON), &jwk)
 	test.AssertNotError(t, err, "Failed to unmarshal test key")
+	validated := time.Now().Round(0).UTC()
 	chall := core.Challenge{
 		Type:                     core.ChallengeTypeDNS01,
-		Status:                   core.StatusPending,
+		Status:                   core.StatusValid,
 		Token:                    "asd",
 		ProvidedKeyAuthorization: "keyauth",
+		Validated:                &validated,
```

Comment:
`test.AssertDeepEquals(t, recon, chall)` on line 70 compares the challenge as crafted on lines 56-61 with the challenge after being passed to a pb and back again on lines 64 and 68. Before I had changed to using *time.Time and was using time.Time it definitely made the difference to add the Round(0) function between this test passing and failing. I haven't been able to get it to fail without the Round(0) since changing the validated field to a pointer to time.Time, but didn't think it would hurt to keep it in there.

An easy demonstration to show that it is being tested, is to strip .UTC() in a test environment that is not set to UTC. That will make the resulting time passed in be compared to a different timezone and fail the test.